### PR TITLE
lint should fail when deprecated code ise used

### PR DESCRIPTION
**Pull request by StojanovicMilos**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
     "@typescript-eslint/eslint-plugin": "5.51.0",
     "@typescript-eslint/parser": "5.51.0",
     "eslint": "8.33.0",
```
Comment (typhoon41) [inline package.json:43]: Please revert this change. This is the wrong package.json. You need to add it to projects/eslint-config one.
On top of that, you need to add this to both dependencies and devDependencies.

Comment (StojanovicMilos) [reply package.json]: Done

**Review by typhoon41 — APPROVED** [review]
